src/pages/gallery/gallery.tsx

```diff
@@ -252,7 +252,6 @@ export default function Gallery({ games, tags }: { games: GameMetadata[], tags:
 							) : null}
 
 							<img
-								src={`gallery/${game.img}`}
```

Comment:
why delete this? will images still show up in gallery?

Reply:
Yep, as stated in #1579 and with the following code block, we can determine that the image data is now injected via base64 on the loadThumbnail function as to a png. The png link errors because it points to the game editor. We can also verify it using the vercel preview.
https://github.com/hackclub/sprig/blob/main/src/pages/gallery/gallery.tsx#L108-L116